public/application.js

```diff
@@ -15,62 +14,45 @@ function humanVersusComputer() {
         dataType: "json"
     });
     request.then((response) => {
-        computerTurn(response);
-        makeHumanMove(response);
-      });
+      putMarkersDown(response);
+      if (gameIsOver(response)) {
+        gameResult(response);
+      };
+    });
   });
 };
 
-function makeHumanMove(response) {
-  if (response.valid === true) {
-    $('#' + response.spot).empty();
-    $('#' + response.spot).append(response.marker);
-    endGame(response);
+function gameIsOver(response) {
+  if (response.over) {
+    return true;
   } else {
-    alert("Invalid move");
+    return false;
   };
-}
+};
 
-function humanVersusHuman() {
-  $("td").click(function(e) {
-    e.preventDefault();
-    let request = $.ajax({
-        type: "PUT",
-        url: "/game",
-        data: {
-            space: $(this).attr('id'),
-        },
-        dataType: "json"
-    });
-    request.then((response) => {
-        makeHumanMove(response);
-        endGame(response);
-      });
+function putMarkersDown(response) {
+  response.moves.forEach(move => {
+    if (move.valid == true) {
```

Comment:
always use triple  equals in javascript: https://stackoverflow.com/questions/523643/difference-between-and-in-javascript

Reply:
also, what should happen if the move isn't valid?